languageserver/integration/config.go

```diff
@@ -47,30 +47,30 @@ type AccountPrivateKey struct {
 //
 // Returns an error if any fields are missing or malformed.
 //
-func configFromInitializationOptions(opts interface{}) (conf Config, err error) {
+func configFromInitializationOptions(opts interface{}) (conf Config, emulatorState float64, err error) {
 	optsMap, ok := opts.(map[string]interface{})
 	if !ok {
-		return Config{}, errors.New("invalid initialization options")
+		return Config{}, 1 , errors.New("invalid initialization options")
```

Comment:
You can make this line and the similar ones below more self-explaining by using the constants you have defined:

```suggestion
		return Config{}, EmulatorOffline, errors.New("invalid initialization options")
```